plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -673,20 +678,4 @@ public function add_product_template( $args ) {
 		}
 		return $args;
 	}
-
-	/**
-	 * Sets the current screen to the block editor if a wc-admin page.
-	 */
-	public function set_current_screen_to_block_editor_if_wc_admin() {
-		$screen = get_current_screen();
-
-		// phpcs:ignore Squiz.PHP.CommentedOutCode.Found
-		// (no idea why I need that phpcs:ignore above, but I'm tired trying to re-write this comment to get it to pass)
-		// we can't check the 'path' query param because client-side routing is used within wc-admin,
-		// so this action handler is only called on the initial page load from the server, which might
-		// not be the product edit page (it mostly likely isn't).
-		if ( PageController::is_admin_page() ) {
-			$screen->is_block_editor( true );
```

Comment:
I wonder if we want to keep those Storefront styles for the iframe editor and the preview area used in the description.  What do you think?

If so, the alternative fix for this would be to remove `editor-styles-wrapper` from our main editor and only wrap this around the preview/iframe editor.

Reply:
Removing `editor-styles-wrapper` from the main editor would prevent these styles from being used in our main editor... assuming that the plugin/theme scoped their CSS to use that (which it should, at a minimum).

But, removing `editor-styles-wrapper` wouldn't prevent the assets from being loaded, which seems wasteful. And, other assets, such as JS, could be enqueued, which might result in incorrect behavior.

I think the most robust approach is to prevent these assets from being loaded at all for our main editor.

Reply:
I think you guys are both missing a big point here. There is a huge legacy of people using other themes and plugins where we would like the benefit of having the block editor enabled for customizing product layouts, but we are not using FSE. In such case, it seems that a more logical solution to this problem that has been created of forcing the loading of core block assets into the product page layouts, would be to:

1) Provide a WooCommerce settings options (similar to disabling ajax or redirection to cart) so that we can choose for ourselves whether we want WC to load all this extra baggage of the blockified product (vs. doing the layout ourselves with other tools via themes/plugins that are not FSE)

2) Making a filter that is pluggable/filterable here that allows us to use the current core block loading filter without it causing the present problem of loading core blocks twice because of what you're trying to improve here (presumably for FSE it seems...as there doesn't seem to be a lot of consideration of all the legacy plugins and themes in anyone's conversation beyond "storefront", which is hardly one of the most popular methods for customizing product layouts in the current marketplace)

Either way, it would be greatly appreciated if there could be some consideration and admin control "before" these aggressive modifications are approved, considering how much is at stake for folk who are operating their livelihood via WooCommerce installations...where each of these updates have significant consequences (ie: break the ability to purchase a product)

thanks!
spence